samples/maps/demo/flowmap/demo.js

```diff
@@ -0,0 +1,142 @@
+(async () => {
+    const topology = await fetch(
+        'https://code.highcharts.com/mapdata/custom/europe.topo.json'
+    ).then(response => response.json());
+
+    Highcharts.mapChart('container', {
+        chart: {
+            map: topology
+        },
+
+        title: {
+            text: 'Highmaps basic lat/lon demo'
+        },
+
+        subtitle: {
+            text: 'Demo of Highcharts map'
+        },
+
+        mapNavigation: {
+            enabled: true
+        },
+
+        series: [{
+            name: 'Basemap',
```

Comment:
This shows in the legend, but doesn't make sense. Probably better to just set `legend.enabled` to false.